graphene/src/box_.rs

```diff
@@ -10,7 +10,7 @@ impl Box {
     #[doc(alias = "get_vertices")]
     pub fn vertices(&self) -> [Vec3; 8] {
         unsafe {
-            let mut out: [ffi::graphene_vec3_t; 8] = std::mem::uninitialized();
+            let mut out: [ffi::graphene_vec3_t; 8] = std::mem::zeroed();
```

Comment:
Considering that we "fill" the array just below, is there any point in filling it with zeroes? Performance wise, it doesn't seem like a good idea...

Reply:
It's not UB like before :)

Reply:
The content is all initialized, but then we call a function which might put random data, so back to square one. In my opinion, this is not an improvement in this context.

Reply:
There's nothing about opinions here. The old code is UB because you are not allowed to get a reference to uninitialized memory. The new code is fine because it initializes it with zeroes (which is a valid value in this case!) before creating a reference that is passed to the C function as a pointer.

What the C functions does or doesn't do is irrelevant in this context.

Reply:
Now that I think about it: why aren't we using `MaybeUninit` here?

Reply:
Ok nevermind, I thought it was an array of structs whereas it's an array of pointers.